paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -109,6 +110,32 @@ def describe_instances(self, instance_ids, region=None, instance_filters=None):
         instances = [instance for reservation in instance_reservations for instance in reservation]
         return instances
 
+    def describe_instance_status(self, instance_ids, region=None, instance_filters=None):
+        """This wraps ec2.describe_instance_status and catches instance not
+        found errors. It returns a list of instance description
+        dictionaries.  Optionally, a filter can be passed through to
+        the ec2 call
+
+        :param instance_ids: a list of instance ids, [] means all
+        :param instance_filters: a list of ec2 filters
+        :param region to connect to ec2
+        :returns: a list of instance description dictionaries"""
+        if not instance_filters:
+            instance_filters = []
+        ec2_client = boto3.client('ec2', region_name=region)
+        try:
+            instance_descriptions = ec2_client.describe_instance_status(
```

Comment:
We didn't need this call before right? But I guess it makes it possible to do the "filtering" outside of `PaastaAwsSlave`?